src/db.js

```diff
@@ -0,0 +1,15 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize(
+  process.env.DB_DATABASE,
+  process.env.DB_USERNAME,
+  process.env.DB_PASSWORD,
+  {
+    host: process.env.DB_HOST,
+    dialect: 'postgres',
+  },
```

Comment:
There is an extra comma after the configuration object. This is a syntax error in JavaScript and should be removed.

Reply:
it is not a syntax error, it is just a trailing comma, it is required by the task eslint configuration